allow installing just the cmake conf
we build the capi ourselves with the same stuff
--
--- a/crates/c-api/CMakeLists.txt
+++ b/crates/c-api/CMakeLists.txt
@@ -115,9 +115,9 @@
     USES_TERMINAL_BUILD TRUE
     BINARY_DIR ${CMAKE_CURRENT_SOURCE_DIR}/artifact
     BUILD_ALWAYS ${WASMTIME_ALWAYS_BUILD}
-    BUILD_BYPRODUCTS ${WASMTIME_SHARED_FILES} ${WASMTIME_STATIC_FILES})
+    BUILD_BYPRODUCTS ${WASMTIME_SHARED_FILES} ${WASMTIME_STATIC_FILES}
+    EXCLUDE_FROM_ALL true)
 add_library(wasmtime INTERFACE)
-add_dependencies(wasmtime wasmtime-crate)
 
 if(BUILD_SHARED_LIBS)
     if(NOT WASMTIME_TARGET MATCHES "windows")
@@ -153,8 +153,6 @@
         FILES_MATCHING REGEX "\\.hh?$")
 install(FILES ${WASMTIME_GENERATED_CONF_H}
         DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/wasmtime)
-install(FILES ${WASMTIME_SHARED_FILES} ${WASMTIME_STATIC_FILES}
-        DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
 if(WASMTIME_TARGET MATCHES "darwin")
   # Postprocess the macOS dylib a bit to have a more reasonable `LC_ID_DYLIB`
